.gitignore

```diff
@@ -48,6 +48,31 @@ yarn-error.log*
 node_modules
 
 
+# CDK asset staging directory
```

Comment:
Not sure what happened here since it looks like duplicated everything in this addition to the `gitignore`